packages/loader/container-loader/src/container.ts

```diff
@@ -1942,6 +1942,7 @@ export class Container
 				this.lastVisible !== undefined ? performance.now() - this.lastVisible : undefined,
 			checkpointSequenceNumber,
 			quorumSize: this._protocolHandler?.quorum.getMembers().size,
+			audienceSize: this._protocolHandler?.audience.getMembers().size,
```

Comment:
Making sure we know the limitations of this: Audience will contain stale content on connecting and "read" connected events (I believe) as we reset audience and populate it (via processing signals, including initial state) after raising "connect" event.
It's not very obvious what state is reported here (i.e. I need to chase code to remind myself when we reset audience and when we populate it). I worry that people looking at this telemetry will make wrong conclusions not understanding limitations here.

Reply:
> When an inactive / sweepReady object usage error is logged in a non-summarizer client, if it was the only client when it connected, we can draw certain conclusions...

If that's the goal, maybe audience size should be a property on those events instead?

Reply:
Also I think that data is "correct" on disconnect and "write" connect events. So maybe we do not write it in other cases not to confuse users?
Note that eventually (when I have time :)) "read" and "write" "connected" events should be identical RE these properties, so I think it's fine exposing this property on connected and disconnected events, but dropping it on catchingUp event.